src/Scene/Scene.js

```diff
@@ -126,32 +129,8 @@ define('Scene/Scene', [
         return this.size;
     };
 
-    /**
-     *
-     * @returns {undefined}
-     */
-    Scene.prototype.quadTreeRequest = function(quadtree, process){
-
-        this.browserScene.browse(quadtree,this.currentCamera(), process, SUBDIVISE);
-        this.managerCommand.runAllCommands().then(function()
-            {
-                if (this.managerCommand.isFree())
-                {
-                    this.browserScene.browse(quadtree,this.currentCamera(), process, SUBDIVISE);
-                    if (this.managerCommand.isFree()){
-                        this.browserScene.browse(quadtree,this.currentCamera(), process, CLEAN)
-                        this.viewerDiv.dispatchEvent(event);
-
-                    }
-                }
-
-            }.bind(this));
-
-        this.renderScene3D();
-
-    };
-
     Scene.prototype.realtimeSceneProcess = function() {
+        // rename sceneUpdate/layerUpdate?
```

Comment:
Indeed `realtimeSceneProcess` isn't clear. `layersUpdate` ?
Probably worth doing it in this PR since you're cleaning the update loop.